HttpClient_Instapush/app/application.cpp

```diff
@@ -45,15 +45,17 @@ class InstapushApplication : protected HttpClient
 
 		DynamicJsonBuffer jsonBuffer;
 		JsonObject& root = jsonBuffer.createObject();
-		root["event"] = event.c_str();
```

Comment:
Sming support storing String classes in JSON memory. Without this feature very hard to control living time of objects. It's why addCopy implemented in our ArduinoJson fork.

Related issue: https://github.com/SmingHub/Sming/issues/59
(solved in last Sming versions)